tests/core/ast_transforms/test_convert_to_jit.py

```diff
@@ -16,23 +16,25 @@ def test_doesnt_convert_math_builtins():
 
 def test_doesnt_convert_jitted_functions():
     @jit
-    def jit_func():
+    def jit_func():     # pragma: no cover
         return 5
 
     assert convert_to_jit(jit_func) is jit_func
+    assert jit_func() == 5
```

Comment:
I don't like the pragma too much, but it strikes me as necessary evil. Some JIT code will simply be unreachable forever, and we need a mechanism to ignore such constructs.